pages/_app.js

```diff
@@ -16,9 +16,22 @@ export default function App({ Component, pageProps }) {
   const [plants, setPlants] = useLocalStorageState("plants", {
     defaultValue: initialPlants,
   });
-
-  
-
+/*---------------------------------------------------------------------- */
+  const [randomTip, setRandomTip] = useState("");
```

Comment:
`randomTip` is initialized as an empty string, but it is expected to hold an object with properties (e.g., randomTip.title, randomTip.imageURL).

This can cause runtime errors if the component tries to access properties of `randomTip` before it is populated. 😖

Consider to nitialise `randomTip` as `null` or provide a fallback object that matches the expected structure.

```suggestion
  const [randomTip, setRandomTip] = useState(null);
```